core_lib/src/interface/timelinecells.cpp

```diff
@@ -171,6 +216,128 @@ bool TimeLineCells::didDetachLayer() const {
     return abs(getMouseMoveY()) > mLayerDetachThreshold;
 }
 
+void TimeLineCells::showCameraMenu(QPoint pos)
+{
+    int frameNumber = getFrameNumber(pos.x());
+    pos = this->mapToGlobal(pos);
+
+    Layer* curLayer = mEditor->layers()->currentLayer();
+    Q_ASSERT(curLayer);
+    // only show menu if on camera layer
+    if (curLayer->type() != Layer::CAMERA)
+    {
+        return;
+    }
+
+    int nextFrame = curLayer->getNextKeyFramePosition(frameNumber);
+
+    QString interpolateAt = tr("Interpolate frames at: %1");
+    QString interpolateFrom = tr("Interpolate frames from: %1 to %2");
+    QString clearMovementDesc = tr("Clear interpolation on: %1");
+
+    if (mEasingMenu == nullptr)
+    {
+        mEasingMenu = new QMenu(this);
+        mInterpolationMenu = new QMenu(mEasingMenu);
+        mHoldAction = new QAction(mEasingMenu);
+
+        QString easeInDesc = tr("Ease-in");
+        QString easeOutDesc = tr("Ease-out");
+        QString slowDesc = tr("Slow");
+        QString normalDesc = tr("Normal");
+        QString quickDesc = tr("Quick");
+        QString fastDesc = tr("Fast");
+        QString fasterDesc = tr("Faster");
+        QString fastestDesc = tr("Fastest");
+        QString circleDesc = tr("Circle-based");
+        QString otherDesc = tr("Other");
+        QString linearDesc = tr("Linear interpolation");
+
+        mInterpolationMenu = mEasingMenu->addMenu(interpolateFrom.arg(QString::number(frameNumber), QString::number(nextFrame)));
+
+        QMenu* subSine  = mInterpolationMenu->addMenu(slowDesc);
+        QMenu* subQuad  = mInterpolationMenu->addMenu(normalDesc);
+        QMenu* subCubic = mInterpolationMenu->addMenu(quickDesc);
+        QMenu* subQuart = mInterpolationMenu->addMenu(fastDesc);
+        QMenu* subQuint = mInterpolationMenu->addMenu(fasterDesc);
+        QMenu* subExpo  = mInterpolationMenu->addMenu(fastestDesc);
+        mInterpolationMenu->addSeparator();
+        QMenu* subCirc  = mInterpolationMenu->addMenu(circleDesc);
+        QMenu* subOther = mInterpolationMenu->addMenu(otherDesc);
+
+        QString descOne = QString("%1 %2");
```

Comment:
Despite I was the one suggesting doing it this way, this will not work for various languages where the order is different.. so I've created a small PR to revert it.